_data/share.yml

```diff
@@ -18,7 +18,7 @@ platforms:
   #
   # - type: Linkedin
   #   icon: "fab fa-linkedin"
-  #   link: "https://www.linkedin.com/sharing/share-offsite/?url=URL"
+  #   link: "https://www.linkedin.com/feed/?shareActive=true&text=TITLE%20URL"
```

Comment:
Let's go with the linkedin URL.

![Screenshot 2025-03-26 at 23 50 57](https://github.com/user-attachments/assets/435b7f4f-90d7-441e-9370-31b4c2fd5734)


```suggestion
  #   link: "https://www.linkedin.com/feed/?shareActive=true&shareUrl=URL"
```